experimental/examples/bubblebench/editable-shared-tree/src/bubble.ts

```diff
@@ -4,10 +4,10 @@
  */
 
 import { IBubble } from "@fluid-example/bubblebench-common";
-import { BubbleTreeProxy } from "./schema";
+import { Bubble } from "./schema";
 
-export class Bubble implements IBubble {
-	constructor(public readonly bubbleTreeProxy: BubbleTreeProxy) {}
+export class BubbleWrapper implements IBubble {
+	constructor(public readonly bubbleTreeProxy: Bubble) {}
```

Comment:
Can we elide this wrapper, or does Bubble not satisfy IBubble?

Reply:
We could, but we choose not to.

In the implementation of this app, we specifically choose to not make architectural simplifications based on the fact that some of the schema based interfaces happen to be API compatible with some of the ones the app defined. In general you can't assume you will get this lucky, and real apps wouldn't want to rely on it either unless the app controlled the APIs and could ensure they kept exact alignment in the future, and individual bubble bench implementations shouldn't assume that.